ansible/maintenance/deploy-tasks-container.yml

```diff
@@ -18,6 +18,10 @@
   - name: Tell tasks containers to drain and restart
     command: pkill -ex cockpit-tasks
 
+  - name: Clean up old container images
+    # this will fail due to at least one used image, but it's opportunistic anyway
+    shell: podman rmi --all || true
```

Comment:
So, even if this would remove all images, that would still be correct, right? We don't rely on the current image being protected by being "in use". Yep, just talking to myself...

Reply:
No, wait.  We _do_ pull the new image before this. Shouldn't this be the other way around?

Reply:
Doesn't matter much -- at that point the old containers are still running, so they will keep the *current* image busy. The point is to remove the older ones. Keeping the current and new images around is fine, and even useful in case we want to quickly revert.